package.json

```diff
@@ -19,6 +19,19 @@
   },
   "homepage": "https://github.com/NelsonAraujo93/toDoList#readme",
   "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "css-loader": "^6.7.3",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.27.5",
+    "hint": "^7.1.3",
+    "sass": "^1.57.1",
+    "sass-loader": "^13.2.0",
+    "style-loader": "^3.3.1",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
     "webpack": "^5.75.0",
     "webpack-cli": "^5.0.1"
   },
```

Comment:
- [ ] As requested by the previous reviewer, your package.json should have scripts like `start` to enable you to start the project with npm start and enable live reloading, please see [webpack documentation](https://webpack.js.org/guides/development/#using-webpack-dev-server) to see how to set it up 👍🏿. For now, I can't start your project running `npm start` as you can see below :point_down: 
![Screenshot from 2023-01-18 19-16-22](https://user-images.githubusercontent.com/92303891/213274781-a3dbc46a-38df-4a32-9211-e9185e1b9c59.png)